cscs-checks/cuda/cuda_memtest_check.py

```diff
@@ -0,0 +1,48 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class FlexibleCudaMemtest(rfm.RegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.valid_systems = ['daint:gpu', 'dom:gpu']
+        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
+                                    'PrgEnv-intel', 'PrgEnv-pgi']
```

Comment:
I don't think it make sense to run this test will all the programming environments. Just pick one.